### DISCUSSION TITLE
Cors functionality not working on Ory Hydra version 2.x

### DISCUSSION THREAD

**Discussion by rtiwari2022** [ory/hydra#3438] [Q&A] [answered]
Hi Team,

 We have deployed all Ory hydra version 2.x (Currently on 2.0.3). However CORS feature is not working as it used to work on version 1.11.10.  Was there any change made to the CORS implementation as part of the 2.x releases ? Kindly please suggest the way how we can enable cors in order to make it work properly in this version. The configuration settings has been followed as mentioned in the documentation however no luck in getting CORS working.

Looking for your help and suggestion on the same.

**Comment by rtiwari2022** [comment]
Can some one please suggest on the same.

**Comment by asterix11** [comment]
I agree with @rtiwari2022 that there may be some flaws. Please review this.

**Comment by aeneasr** [comment, marked as answer]
Hi, I believe a bugfix for this is available on master. Release is scheduled for end of this month
